//package com.sincetimes.redwar.game.comnunicate.protocol;

import "MsgPlanet.proto";

message SpaceStarInfo
{
	optional string name = 1;
	optional string url  = 2;
	optional int32  xp = 3;
	optional int32 score = 4;
	optional string sku = 5;
	optional int32 planetId = 6;
	optional int32 HQLevel = 7;
	optional int32 type = 8;
	optional int32 capital = 9;
	optional int32 damageProtectionTimeLeft = 11;
	optional int32 tutorialCompleted = 12;
	optional int32 isOnline = 13;
	optional string accountId = 14;
	optional int32 levelBasedOnScore = 15;
}

message RseQueryStarInfo
{
	optional int32 ret = 1;
	repeated SpaceStarInfo spaceStarInfo = 2;
	optional int32 type = 3;
	optional string sku = 4;
	optional int32 starId = 5;
	optional int32 name = 6;
}
